packages/ckeditor5-theme-lark/theme/ckeditor5-ui/components/formheader/formheader.css

```diff
@@ -7,13 +7,14 @@
 
 :root {
 	--ck-form-header-height: 44px;
+	--ck-color-form-header-border: var(--ck-color-base-border);
 }
 
 .ck.ck-form__header {
 	padding: var(--ck-spacing-small) var(--ck-spacing-large);
 	height: var(--ck-form-header-height);
 	line-height: var(--ck-form-header-height);
-	border-bottom: 1px solid var(--ck-color-base-border);
+	border-bottom: 1px solid var(--ck-color-form-header-border);
```

Comment:
`FormHeader` could be used in many places, so there is currently no option to override header styles specifically for dialogs. I'm wondering if this is the expected result.